src/OceanSeaIceModels/InterfaceComputations/assemble_net_fluxes.jl

```diff
@@ -81,8 +95,8 @@ end
         ρτx = atmos_ocean_fluxes.x_momentum[i, j, 1]    # zonal momentum flux
         ρτy = atmos_ocean_fluxes.y_momentum[i, j, 1]    # meridional momentum flux
 
-        Qio = sea_ice_ocean_fluxes.heat[i, j, 1]
-        Jˢio = sea_ice_ocean_fluxes.salt[i, j, 1]
+        Qio  = get_ice_heat_flux(i, j, 1, grid, sea_ice_ocean_fluxes)[i, j, 1]
+        Jˢio = get_ice_salt_flux(i, j, 1, grid, sea_ice_ocean_fluxes)[i, j, 1]
```

Comment:
just remember to give some weight to the value of refactoring if that would help clean things up